src/lib/device-online-state/index.ts

```diff
@@ -0,0 +1,282 @@
+import * as Bluebird from 'bluebird';
+declare global {
```

Comment:
Does this literally apply globally or it is limited to file-scope? It should be moved elsewhere if so.

Reply:
I am not sure @dfunckt I will try to find out, but this is the only way I found to make the RSMQ library work with Bluebird